src/zepben/eas/client/eas_client.py

```diff
@@ -262,7 +262,22 @@ async def async_run_hosting_capacity_work_package(self, work_package: WorkPackag
                             } if work_package.results_config.metrics_config is not None else None
                         } if work_package.results_config is not None else None,
-                        "qualityAssuranceProcessing": work_package.quality_assurance_processing if work_package.quality_assurance_processing is not None else None
+                        "qualityAssuranceProcessing": work_package.quality_assurance_processing if work_package.quality_assurance_processing is not None else None,
+                        "generatorConfig": {
+                            "pFactorBaseExports": work_package.generator_config.p_factor_base_exports if work_package.generator_config.p_factor_base_exports is not None else None,
+                            "pFactorBaseImports": work_package.generator_config.p_factor_base_imports if work_package.generator_config.p_factor_base_imports is not None else None,
+                            "pFactorForecastPv": work_package.generator_config.p_factor_forecast_pv if work_package.generator_config.p_factor_forecast_pv is not None else None,
+                            "fixSinglePhaseLoads": work_package.generator_config.fix_single_phase_loads if work_package.generator_config.fix_single_phase_loads is not None else None,
+                            "maxSinglePhaseLoad": work_package.generator_config.max_single_phase_load if work_package.generator_config.max_single_phase_load is not None else None,
+                            "fixOverloadingConsumers": work_package.generator_config.fix_overloading_consumers if work_package.generator_config.fix_overloading_consumers is not None else None,
+                            "maxLoadTxRatio": work_package.generator_config.max_load_tx_ratio if work_package.generator_config.max_load_tx_ratio is not None else None,
+                            "maxGenTxRatio": work_package.generator_config.max_gen_tx_ratio if work_package.generator_config.max_gen_tx_ratio is not None else None,
+                            "fixUndersizedServiceLines": work_package.generator_config.fix_undersized_service_lines if work_package.generator_config.fix_undersized_service_lines is not None else None,
+                            "maxLoadLineRatio": work_package.generator_config.max_load_line_ratio if work_package.generator_config.max_load_line_ratio is not None else None,
+                            "collapseLvNetworks": work_package.generator_config.collapse_lv_networks if work_package.generator_config.collapse_lv_networks is not None else None,
+                            "feederScenarioAllocationStrategy": work_package.generator_config.feeder_scenario_allocation_strategy if work_package.generator_config.feeder_scenario_allocation_strategy is not None else None,
+                            "includeCalibrationMeters": work_package.generator_config.include_calibration_meters if work_package.generator_config.include_calibration_meters is not None else None,
```

Comment:
all these inline ifs don't actually do anything - should just be:
`"includeCalibrationMeters": work_package.generator_config.include_calibration_meters`